.github/workflows/diff_shades.yml

```
@@ -48,7 +49,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        include: ${{ fromJson(needs.configure.outputs.matrix )}}
+        include: ${{ fromJson(needs.configure.outputs.matrix ) }}
```

```suggestion
        include: ${{ fromJson(needs.configure.outputs.matrix) }}
```
Does prettier not update this part?

---

Prettier does not format Actions syntax and formats them the same as other .yml files - the space before the `}}` was something I noticed and added manually, but I didn't see the space before the `)`.